pom.xml

```diff
@@ -203,8 +203,9 @@
                     <version>${maven.release.plugin.version}</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <preparationGoals>clean verify install</preparationGoals>
+                        <preparationGoals>clean install</preparationGoals>
                         <pushChanges>false</pushChanges>
+                        <arguments>-Prelease ${arguments}</arguments>
```

Comment:
Hmm, I don't think we need to hold off on Javadoc generation but I believe we should address that before the next Alliance release and clearly mark APIs internal/experimental if that is truly the intent.